stock_orderpoint_manual_procurement/models/procurement.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ProcurementRule(models.Model):
+    _inherit = 'procurement.rule'
+
+    def _prepare_mo_vals(self, product_id, product_qty,
```

Comment:
I just realized that this is a method added in `mrp` module. Maybe we need to move this part to a glue module, something like `mrp_orderpoint_manual_procurement`. Opinions? @jbeficent